tasks/compile-translations-po-to-json.js

```diff
@@ -82,7 +83,7 @@ function compileTranslationsPoToJson(grunt) {
             return;
         }
 
-        var po2json = path.join(clientCoreRoot, 'node_modules/gettext.js/bin/po2json');
+        var po2json = `${getModuleDir('gettext.js')}/bin/po2json`;
```

Comment:
`npx po2json` should handle it no matter where it is

Reply:
```
execSync(
            `npx gettext.js/bin/po2json ${poFile} ${jsonFile}`,
            {stdio: 'inherit'}
        );
```

> ENOENT: no such file or directory, stat 'gettext.js/bin/po2json'

Reply:
It should be `npx po2json ...`

Reply:
my bad. It still doesn't work though

```
execSync(
            `npx po2json ${poFile} ${jsonFile}`,
            {stdio: 'inherit'}
        );
```

```
grunt gettext:compile
Running "gettext:compile" task
Warning: Cannot read property 'match' of null Use --force to continue.

Aborted due to warnings.
```